kde-plasma/plasma-pa/plasma-pa-5.27.7.ebuild

```diff
@@ -31,6 +31,5 @@ DEPEND="
 RDEPEND="${DEPEND}
 	>=dev-qt/qtquickcontrols2-${QTMIN}:5
 	>=kde-frameworks/kirigami-${KFMIN}:5
-	x11-themes/sound-theme-freedesktop
```

Comment:
Changes to RDEPEND require a new revision. For the purposes of reviewing the PR, it's fine if you do these as `git mv`.